docs/release/release-note-0.3.0.md

```diff
@@ -0,0 +1,30 @@
+# Release Note 0.3.0
+
+## ONE-vscode
+
+### OneExplorerView
+
+- Support more commands
+  - [Open with Text Editor] Command
```

Comment:
```suggestion
  - [Open with Text Editor] command
```

Reply:
I think `command` is a common noun. Would lowercase be correct..?

Reply:
I will fix it.